book/website/project-design/pd-overview/pd-overview-repro.md

```diff
@@ -1,44 +1,68 @@
+It should contain a **recognizable name** for your project, as well as a **declaration of the licensing terms** under which it may be distributed (see the {ref}`OH_licences` chapter).
 
 Documentation should be shared via centralised, findable and accessible platforms.
-It is particularly important to share the vision, mission and milestones clearly. 
+
+## Documenting project plans and processes
+ 
+Document the **current state**, **ongoing development**, and/or any **future plans** for the project.
+Add information on available resources and recommended practices to ensure everyone is on the same page (literally!).
+It may also be important to make it clear what your project is not meant to be.
+
+In addition to the above, it is interesting to document the *principal need* for your project, especially for software and hardware project: what problem are you planning to solve ?
+That includes a **problem description** of whatever issue sparked the project, a **functional description** of how your project is meant to address it, and a **context description** of the users and environment your project is targeting. 
+
+It is particularly important to share the **vision, mission and milestones** clearly. 
 Provide sufficient information for what the expected outcomes and deliverables are.
 
+
 Provide overarching as well as short-term goals and describe expected outcomes to help contributors move away from focusing on a single idea of the feature.
 Describe the possible expansion of features in pre-determined and agreed on ways at stages beyond the initial implementation.
 
-**Project's "Who-is-Who"**:
+## Project's "Who-is-Who"
+
+Include a **list of contributors**, **contribution guidelines**, and information on **contact points** where to ask for help.
+In addition, 
+create a directory to highlight the different stakeholders with their roles in the project, keys skills, interests and contact information (when possible).
 
-Create a directory to highlight the different stakeholders with their roles in the project, keys skills, interests and contact information (when possible).
 Describe what opportunities for collaboration different members will have.
 When possible, such as in an open source project, provide these details for those outside the current group, especially when you want to encourage people outside the project to be involved.
+Communicate the work culture that you want to promote and policies that ensures the safety and security of both your data and people.
+
+
+Provide resources on ways of working to ensure fair participation of contributors who collaborate on short- and long-term milestones within the project.
+It reduces or addresses concerns about the project's progress towards meeting goals and prevents potential fallout between project contributors.
```

Comment:
`prevents` -> `may mitigate`

Reply:
original text, I would keep it as it is, as the changes are not crucial to the discourse (any concerns are meant I think)